go_lib/dependency/cr/cr.go

```diff
@@ -99,6 +100,28 @@ func (r *client) Image(tag string) (v1.Image, error) {
 	)
 }
 
+func (r *client) ListTags() ([]string, error) {
+	var nameOpts []name.Option
+	if r.options.useHTTP {
+		nameOpts = append(nameOpts, name.Insecure)
+	}
+
+	imageOptions := make([]remote.Option, 0)
+	if !r.options.withoutAuth {
```

Comment:
Can we use true condition ? something like r.options.withAuth ?

Reply:
Don't think so, we have to refactor a lot of code then